packages/client/src/components/CreatePost.js

```diff
@@ -46,6 +47,7 @@ function CreatePost() {
 
   return (
     <>
+      <Navbar isAdmin={isAdmin}></Navbar>
```

Comment:
At some point of time we should probably lift these up to the app component, no reason for rendering these differently so many times.

I'll make a ticket for that.